hw/drivers/ram/spiram/src/spiram.c

```diff
@@ -102,7 +102,7 @@ spiram_write_enable(struct spiram_dev *dev)
 int
 spiram_read(struct spiram_dev *dev, uint32_t addr, void *buf, uint32_t size)
 {
-    uint8_t cmd[5] = { SPIRAM_READ };
+    uint8_t cmd[6] = { SPIRAM_READ };
```

Comment:
Does it make sense to break the limit down into 3/4 byte addressing + max command length or something ?

Reply:
Given these parameters don't change on the fly, splitting these into 
```
(1 + MYNEWT_VAL(SPIRAM_0_ADDRESS_BYTES) + MYNEWT_VAL(SPIRAM_0_DUMMY_BYTES)) 
```
might make this more readable.
@kasjer can we merge this soon

Reply:
Agreed, that does seem like a better approach.

Reply:
we can't, SPIRAM_0_ADDRESS_BYTES is per instance setting not per driver

Reply:
then maybe we use `cmd_size` that uses per instance setting?

Reply:
there is no point doing this unless you aim is to have more flash used by the code, slower execution time and more stack used.

Reply:
```
/* The user is required to specify both based on their config and usage. */

(1 + MYNEWT_VAL(SPIRAM_MAX_ADDRESS_BYTES) + MYNEWT_VAL(SPIRAM_MAX_DUMMY_BYTES))

```

Reply:
I'll take it as a joke.

Reply:
I know its not super intuitive but it is better than 5-6 hardcoded value which seems we have no clue about where it came from.

Reply:
@kasjer  Can you please address this ? Let's get it merged.